### PR TITLE
reviewed saturn0 landing page

### PR DESCRIPTION
**Pull request by orangesamus**

### CODE REVIEW

File: docs/guides/saturn-0/whats-new.mdx [hunk 1]
```diff
@@ -7,42 +7,38 @@ serve as an introduction to some new and exciting changes to the Rocket Pool pro
...
-- Saturn 0 minipools have a 5% base ETH comission, which will be temporarily boosted up to 14% after [Reward Tree Spec v10](https://rpips.rocketpool.net/assets/rpip-62/rewards-calculation-spec), up until the launch of Saturn 1.
-
 ## New Commission Structure (Saturn 0 part 2)
 ::: tip NOTE
 Dynamic Commission will be enabled when v10 of the [Reward Tree Spec](https://rpips.rocketpool.net/assets/rpip-62/rewards-calculation-spec) is implemented. It's currently in development, so keep an eye on [#protocol](https://discord.com/channels/405159462932971535/405163979141545995) 
 announcements in our [Discord Server](https://discord.com/invite/rocketpool) for updates!
 :::
 
-We're excited to announce that newly launched Saturn 0 minipools will feature a commission structure ranging from 5% to 14%.
+We're excited to announce that newly launched Saturn 0 minipools will feature a commission structure ranging from 10% to 14%.
```
Comment (thomaspanf) [inline docs/guides/saturn-0/whats-new.mdx:32]: We have this message in the SN as 5-14: https://github.com/rocket-pool/smartnode/pull/681#event-14740168889

I reckon we should just keep it consistent with the SN?

File: docs/guides/saturn-0/whats-new.mdx [hunk 2]
```diff
@@ -7,42 +7,38 @@ serve as an introduction to some new and exciting changes to the Rocket Pool pro
...
 :::
 
 
 ## ETH only minipools in Saturn 0 
 
-- The RPL requirement for new minipools are a thing of the past! Minipools launched after the inception of Saturn 0 do not require RPL stake to be deployed. 
-Node operators can now create minipools with an upfront cost of 8 ETH. In addition, a node operator may stake RPL to qualify for additional rewards 
-and access voting power within the [Protocol DAO](guides/houston/pdao#the-protocol-dao-pdao). 
+- The RPL requirement for new minipools is a thing of the past! Minipools launched after the inception of Saturn 0 do not require RPL stake to be deployed.
+Node operators can now create minipools with an upfront cost of 8 ETH, which will feature a commission ranging from 10% to 14%. In addition, a node
```
Comment (thomaspanf) [inline docs/guides/saturn-0/whats-new.mdx]: Here also

Comment (haloooloolo) [inline docs/guides/saturn-0/whats-new.mdx]:
```suggestion
Node operators can now create minipools with an upfront cost of 8 ETH, which will feature a commission of 10%. In addition, a node
```
Something like that. If the next sentence talks about additional rewards with RPL separately, this one should probably be just about ETH-only.

File: docs/guides/saturn-0/whats-new.mdx [hunk 3]
```diff
@@ -7,42 +7,38 @@ serve as an introduction to some new and exciting changes to the Rocket Pool pro
...
+- Nodes without any staked RPL will earn 10% commission (5% from the minipool contract, and 5% from the dynamic commission boost).
+- Nodes with staked RPL receive up to an additional 4% dynamic commission boost for staked RPL valued at >= 10% of borrowed ETH.
 
-For a total of up to 14% commission on borrowed ETH. 
+Note that the dynamic commission boost will be reduced to 0% on the 5th reward snapshot after Saturn 1 is released (leaving only the 5% minipool contract commission). This serves to increase the yield advantage of migrating to megapools.
 
 ::: warning WARNING
-Your node must be opted into the [Smoothing Pool](/guides/node/fee-distrib-sp#the-smoothing-pool) to benefit from the dynamic comission boost, but don't worry. The latest Smart Node version `v1.14.0` will inform you 
-to opt into the Smoothing Pool when depositing a minipool if you aren't already opted in. *You new minipools will miss out on additional yield if your node 
+Your node must be opted into the [Smoothing Pool](/guides/node/fee-distrib-sp#the-smoothing-pool) to benefit from the dynamic comission boost. Don't worry, the latest Smart Node version `v1.14.0` will inform you
```
Comment (haloooloolo) [inline docs/guides/saturn-0/whats-new.mdx]: Didn't comment on it initially, but the "don't worry" feels a little patronizing to me. Doesn't fit in well with the style the docs are usually written in, in my opinion.

Comment (orangesamus) [inline docs/guides/saturn-0/whats-new.mdx]:
```suggestion
Your node must be opted into the [Smoothing Pool](/guides/node/fee-distrib-sp#the-smoothing-pool) to benefit from the dynamic comission boost. The latest Smart Node version `v1.14.0` will inform you
```

Comment (orangesamus) [reply docs/guides/saturn-0/whats-new.mdx]: Sure, removed it

File: docs/guides/saturn-0/whats-new.mdx (hunk 1 above)
Comment (orangesamus) [reply docs/guides/saturn-0/whats-new.mdx:32]: I responded in discord: https://discord.com/channels/405159462932971535/922648971351056475/1299160800203902986

File: docs/guides/saturn-0/whats-new.mdx (hunk 2 above)
Comment (orangesamus) [reply docs/guides/saturn-0/whats-new.mdx]: See discord: https://discord.com/channels/405159462932971535/922648971351056475/1299160800203902986

**Pull request merged**